integration/container/copy_test.go

```diff
@@ -64,3 +68,61 @@ func TestCopyToContainerPathIsNotDir(t *testing.T) {
 	err := apiclient.CopyToContainer(ctx, cid, "/etc/passwd/", nil, types.CopyToContainerOptions{})
 	assert.Assert(t, is.ErrorContains(err, "not a directory"))
 }
+
+func TestCopyFromContainerRoot(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType == "windows", "FIXME")
+	defer setupTest(t)()
+
+	ctx := context.Background()
+	apiClient := testEnv.APIClient()
+
+	buildCtx := fakecontext.New(t, "", fakecontext.WithDockerfile(`
+		FROM busybox AS work
+		RUN echo hello > /foo
```

Comment:
need to also test a directory. Could probably just be part of the context so we can have a simple stage with `FROM` + `COPY`. scratch on linux and whatever is the default on windows.